tests/Unit/Console/Command/AdminPromoteCommandTest.php

```diff
@@ -80,15 +80,15 @@ public function promoteExistingNonAdminAccount()
         $this->assertEquals($response, 0);
     }
 
-    protected function createInputInterfaceWithEmail($email)
+    protected function createInputInterfaceWithEmail($email): \Symfony\Component\Console\Input\InputInterface
     {
         $input = Mockery::mock(\Symfony\Component\Console\Input\InputInterface::class);
         $input->shouldReceive('getArgument')->with('email')->andReturn($email);
 
         return $input;
     }
 
-    protected function createOutputInterface()
+    protected function createOutputInterface(): \Symfony\Component\Console\Output\OutputInterface
```

Comment:
Instead of mocking `InputInterface` and `OutputInterface`, we could just use `Symfony\Component\Console\Tester\CommandTester`. I tried, but there's a lot of work required in regard to the application container being injected into the individual commands (rather than injecting dependencies into the commands and then pulling the commands from the container). Because of #618, I think it's not really worth the effort now, but suggest to sort this out anyway.